src/oidcendpoint/oidc/add_on/pkce.py

```diff
@@ -82,7 +103,10 @@ def post_token_parse(request, client_id, endpoint_context, **kwargs):
                 request["code_verifier"], _info["authn_req"]["code_challenge"], _method
```

Comment:
`verify_code_challenge` will return a TokenErrorResponse now, so we should assign its return value to a variable (`request`), otherwise it never fails the verification.